platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java

```diff
@@ -377,4 +413,35 @@ public static <T> Consumer<T> combineConsumers(final List<Consumer<T>> consumers
             }
         };
     }
+
+    /**
+     * Returns a string representation of the given byte count in human readable format.
+     *
+     * @param bytes number of bytes
+     * @return human-readable string representation of the given byte count
+     */
+    public static String byteCountToDisplaySize(final long bytes) {
+        if (bytes < 1024) {
+            return bytes + " B";
+        }
+        final int z = (63 - Long.numberOfLeadingZeros(bytes)) / 10;
+        return String.format("%.1f %sB", (double) bytes / (1L << (z * 10)), " KMGTPE".charAt(z));
+    }
```

Comment:
There is already a platform utility for creating human readable strings out of things like bytes.

`byteCountToDisplaySize(1234)` can be replaced with `new UnitFormatter(1234, UNIT_BYTES).render()`. See `UnitFormatterTests.java` for further examples.